docs/src/cookbooks/cookbook.md

```diff
@@ -462,3 +464,52 @@ ChiselStage.emitVerilog(new Salt)
 ```scala mdoc:verilog
 ChiselStage.emitVerilog(new Salt)
 ```
+
+## Directionality
+
+### How do I strip directions from a bidirectional Bundle (or other Data)?
+
+Given a bidirectional port like a `Decoupled`, you will get an error if you try to connect it directly
+to a register:
+
+```scala mdoc:silent
+import chisel3.util.Decoupled
+class BadRegConnect extends Module {
+  val io = IO(new Bundle {
+    val enq = Decoupled(UInt(8.W))
+  })
+  
+  val monitor = RegNext(io.enq)
+}
+```
+
+```scala mdoc:crash
+ChiselStage.emitVerilog(new BadRegConnect)
+```
+
+While there is no construct to "strip direction" in Chisel3, wrapping a type in `Output(...)` will
+coerce the direction to `Output` (even if the input is bidirectional).
+This will have the desired result when used to construct a Register:
```

Comment:
```suggestion
While there is no construct to "strip direction" in Chisel3, wrapping a type in `Output(...)`
(the default direction in Chisel3) will
remove the directionality information of the individual elements.
This will have the desired result when used to construct a Register:
```

Reply:
> "remove the directionality information of the individual elements"

That's not strictly true, it does set them all to `Output` but reg is fine with that 😬

Reply:
ok editing the above